development/test-log-collector/README.md

```diff
@@ -0,0 +1,62 @@
+# Test Log Collector
+
+## Overview
+
+The purpose of the Test Log Collector is to gather logs from the testing pods and to send them to the appropriate slack channels.
+It is intended to be run after the Octopus ClusterTestSuite, as it uses pods labels from Octopus. Test Log Collector will send logs only from the newest, finalised ClusterTestSuite.
```

Comment:
```suggestion
It is intended to be run after the Octopus ClusterTestSuite, as it uses Pods labels from Octopus. By design, the Test Log Collector sends logs only from the newest, finalized ClusterTestSuite.
```

Reply:
1. pods -> Pods (we capitalize Pods)
2.  will send -> we rather avoid future tenses so I tried to rephrase this part
3. "the" Test Log Collector - only to unify it throughout the text as you used "the" above
4. finalised -> AmE so "z"